tensorflow/compiler/xla/service/elemental_ir_emitter.cc

```diff
@@ -1292,6 +1292,16 @@ StatusOr<llvm::Value*> ElementalIrEmitter::EmitComplexBinaryOp(
                     llvm_ir::EmitComparison(llvm::CmpInst::FCMP_UNE,
                                             EmitExtractImag(lhs_value),
                                             EmitExtractImag(rhs_value), b_));
+	case ComparisonDirection::kGt:
```

Comment:
What happens to other cases? kLt? Less-or-equal? You could also support this by implementing a normalization pass in algebraic_simplifier